File: src/com/prey/beta/actions/PreyBetaActionsRunnner.java
/*******************************************************************************
 * Created by Carlos Yaconi
 * Copyright 2012 Fork Ltd. All rights reserved.
 * License: GPLv3
 * Full license at "/LICENSE"
 ******************************************************************************/
package com.prey.beta.actions;

 


import android.content.Context; 

public class PreyBetaActionsRunnner {

	protected boolean running = false;

	Thread myActionsRunnerThread = null;
	private String cmd;

	public PreyBetaActionsRunnner(String cmd) {
		this.cmd=cmd;
	}

	public void run(Context ctx) {
		this.myActionsRunnerThread = new Thread(new PreyBetaActionsRunner(ctx,cmd));
		this.myActionsRunnerThread.start();

	}
 
}

File: app/src/main/java/com/prey/beta/actions/PreyBetaActionsRunnner.kt
/*******************************************************************************
 * Created by Orlando Aliaga
 * Copyright 2025 Prey Inc. All rights reserved.
 * License: GPLv3
 * Full license at "/LICENSE"
 ******************************************************************************/
package com.prey.beta.actions

import android.content.Context

class PreyBetaActionsRunnner {

    protected var running: Boolean = false
    private var myActionsRunnerThread: Thread? = null
    private var cmd: String? = null

    fun PreyBetaActionsRunnner(cmd: String) {
        this.cmd = cmd
    }

    fun run(ctx: Context) {
        this.myActionsRunnerThread = Thread(PreyBetaActionsRunner(ctx, cmd))
        myActionsRunnerThread!!.start()
    }

}